CRM/Civiquickbooks/Invoice.php

```diff
@@ -658,7 +663,7 @@ protected function mapToAccounts($db_contribution, $accountsID, $SyncToken, $qb_
       }
 
       if ($whereToGetInvoiceNumber == 'civi') {
-        $new_invoice['DocNumber'] = 'Civi-' . $db_contribution['id'];
+        $new_invoice['DocNumber'] = $invoice_prefix . sprintf("%07d", $db_contribution['id']);
```

Comment:
Alternative would be to do `sprintf($invoice_prefix, $db_contribution['id'])` where `$invoice_prefix` is in the format `'Civi-%07d'` (for this example)